### PR TITLE
Translate concepts/policy/limit-range into Japanese

### LINKED ISSUE #22812: Translate concepts/policy/limit-range into Japanese
**This is a Feature Request**

**What would you like to be added**

Translate [concepts/policy/limit-range](https://kubernetes.io/docs/concepts/policy/limit-range) into Japanese.

**Why is this needed**

This page has not been translated.

**Comments**

/language ja
/assign

### PR DESCRIPTION
**Pull request by shuuji3**
Closes #22812.

### CODE REVIEW

**Comment by shuuji3** [comment]
https://deploy-preview-22873--kubernetes-io-master-staging.netlify.app/ja/docs/concepts/policy/limit-range/

File: content/ja/docs/concepts/policy/limit-range.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+- 管理者は、1つの名前空間に1つのLimitRangeを作成します。
+- ユーザーは、Pod、コンテナ、PersistentVolumeClaimsのようなリソースを名前空間内に作成します。
+- `LimitRanger` admissionコントローラーは、計算リソース要求が設定されていないすべてのPodとコンテナに対して、デフォルト値と制限値を強制します。そして、リソースの使用量を追跡し、名前空間内に存在するすべてのLimitRangeで定義された最小値、最大値、割合を外れないことを保証します。
+- LimitRangeの制約を破るようなリソース(Pod、コンテナ、PersistentVolumeClaim)の作成や更新を行うと、APIサーバーへのリクエストがHTTPステータスコード`403 FORBIDDEN`で失敗し、破られた制約を説明するメッセージが返されます。
+- 名前空間内でLimitRangeが`cpu`や`memory`などの計算リソースに対して有効になっている場合、ユーザーはrequestsやlimitsに値を指定しなければなりません。指定しなかった場合、システムはPodの作成を拒否する可能性があります。
+- LimitRangeの制約違反は、PodのAdmissionステージでのみ発生し、Running状態のPodでは発生しせん。
+
+以下は、LimitRangeを使用して作成できるポリシーの例です。
+
+- 8GiBのRAMと16コアのCPUの容量がある2ノードのクラスター上で、名前空間内のPodに対して、CPUには100mのrequestと最大500mのlimitの制約を課し、メモリには200Miのrequestと600Miのlimitの制約を課す。
```
Comment (oke-py) [inline content/ja/docs/concepts/policy/limit-range.md:39]:
```suggestion
- 8GiBのRAMと16コアのCPUの容量がある2ノードのクラスター上で、名前空間内のPodに対して、CPUには100mのrequestと最大500mのlimitの制約を課し、メモリーには200Miのrequestと600Miのlimitの制約を課す。
```

File: content/ja/docs/concepts/policy/limit-range.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+- Spec内のrequestsにcpuやmemoryを指定せずに起動したコンテナに対して、CPUにはデフォルトで150mのlimitとrequestを、メモリにはデフォルトで300Miのrequestをそれぞれ定義する。
```
Comment (oke-py) [inline content/ja/docs/concepts/policy/limit-range.md:40]:
```suggestion
- Spec内のrequestsにcpuやmemoryを指定せずに起動したコンテナに対して、CPUにはデフォルトで150mのlimitとrequestを、メモリーにはデフォルトで300Miのrequestをそれぞれ定義する。
```

File: content/ja/docs/concepts/policy/limit-range.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+LimitRangeに対する競合や変更は、すでに作成済みのリソースに対しては影響しません。
+
+## {{% heading "whatsnext" %}}
+
+より詳しい情報は、[LimitRangerの設計ドキュメント](https://git.k8s.io/community/contributors/design-proposals/resource-management/admission_control_limit_range.md)を参照してください。
+
+制限の使用例については、以下のページを読んでください。
+
+- [名前空間ごとにCPUの最小値と最大値の制約を設定する方法](/docs/tasks/administer-cluster/manage-resources/cpu-constraint-namespace/)。
+- [名前空間ごとにメモリの最小値と最大値の制約を設定する方法](/docs/tasks/administer-cluster/manage-resources/memory-constraint-namespace/)。
```
Comment (oke-py) [inline content/ja/docs/concepts/policy/limit-range.md:53]:
```suggestion
- [名前空間ごとにメモリーの最小値と最大値の制約を設定する方法](/docs/tasks/administer-cluster/manage-resources/memory-constraint-namespace/)。
```

File: content/ja/docs/concepts/policy/limit-range.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+- [名前空間ごとにCPUのRequestとLimitのデフォルト値を設定する方法](/docs/tasks/administer-cluster/manage-resources/cpu-default-namespace/)。
+- [名前空間ごとにメモリのRequestとLimitのデフォルト値を設定する方法](/docs/tasks/administer-cluster/manage-resources/memory-default-namespace/)。
```
Comment (oke-py) [inline content/ja/docs/concepts/policy/limit-range.md:55]:
```suggestion
- [名前空間ごとにメモリーのRequestとLimitのデフォルト値を設定する方法](/docs/tasks/administer-cluster/manage-resources/memory-default-namespace/)。
```

File: content/ja/docs/concepts/policy/limit-range.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+LimitRangeオブジェクトの名前は、有効な[DNSサブドメイン名](/ja/docs/concepts/overview/working-with-objects/names#dns-subdomain-names)でなければなりません。
+
+### Limit Rangeの概要
+
+- 管理者は、1つの名前空間に1つのLimitRangeを作成します。
+- ユーザーは、Pod、コンテナ、PersistentVolumeClaimsのようなリソースを名前空間内に作成します。
+- `LimitRanger` admissionコントローラーは、計算リソース要求が設定されていないすべてのPodとコンテナに対して、デフォルト値と制限値を強制します。そして、リソースの使用量を追跡し、名前空間内に存在するすべてのLimitRangeで定義された最小値、最大値、割合を外れないことを保証します。
+- LimitRangeの制約を破るようなリソース(Pod、コンテナ、PersistentVolumeClaim)の作成や更新を行うと、APIサーバーへのリクエストがHTTPステータスコード`403 FORBIDDEN`で失敗し、破られた制約を説明するメッセージが返されます。
+- 名前空間内でLimitRangeが`cpu`や`memory`などの計算リソースに対して有効になっている場合、ユーザーはrequestsやlimitsに値を指定しなければなりません。指定しなかった場合、システムはPodの作成を拒否する可能性があります。
+- LimitRangeの制約違反は、PodのAdmissionステージでのみ発生し、Running状態のPodでは発生しせん。
```
[the remark this answers is not in the document]
Comment (shuuji3) [reply content/ja/docs/concepts/policy/limit-range.md:35]: たしかにこの方がわかりやすいですね。

**Pull request merged**